styles/common.module.scss

```
@@ -0,0 +1,5 @@
+.flexCenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

We will be using this class various times in the project, so please, extract this class in the `common.scss` under the styles folder.

```suggestion
  @extend .flexCenter;
```